src/components/TeaserEmbedComment/index.js

```diff
@@ -74,17 +75,19 @@ const TeaserEmbedComment = ({ data, liveData, t, Link }) => {
           discussion={discussionContextValue.discussion}
           passHref
         >
-          <div style={{ margin: '10px 0', cursor: 'pointer' }}>
-            <Comment.Body
-              t={t}
-              comment={contentComment}
-              context={
-                contentComment.tags && contentComment.tags[0]
-                  ? { title: contentComment.tags[0] }
-                  : undefined
-              }
-            />
-          </div>
+          <a {...plainLinkRule}>
+            <div style={{ margin: '10px 0' }}>
```

Comment:
Could be one element :P

```suggestion
          <a {...plainLinkRule} style={{ display: 'block', margin: '10px 0' }}>
```